src/widgets/clear-refinements/clear-refinements.tsx

```diff
@@ -116,6 +116,10 @@ const clearRefinements: ClearRefinementsWidget = widgetParams => {
       userCssClasses.disabledButton
     ),
   };
+  const templates = {
+    ...defaultTemplates,
+    ...userTemplates,
+  };
```

Comment:
this also isn't needed, as it gets defaulted in prepareTemplateProps

Reply:
see https://github.com/algolia/instantsearch.js/pull/4785/commits/61ea462ec076c7fd7a38a33f8d0bde9344d72d0a